test/modules/TestResultCoordinator/TestResultCoordinator.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
I think at least you have the changes:
1. handle to build .net core 2.1 and 3.0
2. stylecop
3. reference to ModuleLib

Reply:
Done